### ISSUE TITLE
User management - Group-Based Access Management

### ISSUE THREAD

**Issue by p-liubchenko** [datalust/seq-tickets#2320]
In our project, we provide access to multiple developers, and as the project grows, it is becoming increasingly challenging to manage granular access permissions (especially view filters) for individual users.

It would be highly beneficial to have a feature that allows the creation of user groups, enabling administrators to manage access permissions at the group level. This would streamline user management and improve scalability as the number of users increases.

Thank you for considering this request, and please let me know if additional details or use cases would be helpful.

**Comment by nblumhardt** [comment]
Thanks for raising this, @p-liubchenko. SCIM/SAML IdP integration is working its way up [our roadmap](https://datalust.co/roadmap) and aligns fairly strongly with this. Are you also using an identity provider that you would seek to automate user management by integrating with?

**Comment by p-liubchenko** [comment]
NO, currently we are using integrated identity provider. our company is multitenant. we have 3+ vendors and even though all vendors use M365 it's a bit complicated to configure access for all of them. so we rely on integrated auth provider.
but group management is something will be handy.
